cmd/postgres-reader/main.go

```diff
@@ -147,9 +148,9 @@ func main() {
 
 func newService(db *sqlx.DB, logger *slog.Logger) readers.MessageRepository {
 	svc := postgres.New(db)
-	svc = api.LoggingMiddleware(svc, logger)
+	svc = middleware.LoggingMiddleware(svc, logger)
```

Comment:
To reduce stuttering 

```suggestion
	svc = middleware.Logging(svc, logger)
```